css/app.css

```diff
@@ -321,4 +317,10 @@ input.form-control:focus{
 .monsterName h1 {
     font-family: 'Metal Mania', cursive;
     font-size: 60px;
+    color: #ff4828;
+}
+
+.improve{
```

Comment:
Try to edit existing classes instead of new class. If it is required please provide an appropriate name. It's surely an improvement/fix for a bug but the code should have a meaningful name.

Reply:
I have included changes within using available classes. New PR sent